### PR TITLE
foliate: update to 2.1.0

### LINKED ISSUE #20714: foliate executable has unintuitive name
### System

* xuname:  
  *Void 5.4.29_1 x86_64 GenuineIntel uptodate hold rrrmDFFFFFFFFFFFFFF*
* package:  
  *affected package(s) including the version*

### Expected behavior
run [Foliate](https://johnfactotum.github.io/foliate/") via `foliate` or similar
### Actual behavior
Foliate must be run as `com.github.johnfactotum.Foliate`
### Steps to reproduce the behavior
Install `foliate`

(I'm not sure the best way of renaming the executable in the [package build](https://github.com/void-linux/void-packages/blob/master/srcpkgs/foliate/template). I tried adding:

```
post_install() {
	vbin com.github.johnfactotum.Foliate foliate
}
```
And some similar variants, but didn't have any luck.)

### PR DESCRIPTION
**Pull request by not-chicken**
Closes #20714

### CODE REVIEW

File: srcpkgs/foliate/template [hunk 1]
```diff
@@ -12,3 +12,7 @@ license="GPL-3.0-or-later"
 homepage="https://johnfactotum.github.io/foliate/"
 distfiles="https://github.com/johnfactotum/foliate/archive/${version}.tar.gz"
 checksum=e3f7ab5242aa6d936b66d6a41147bf17c0334d7675b5e38e64829d204fd3712a
+
+post_install() {
+	ln -sv /usr/bin/com.github.johnfactotum.Foliate ${DESTDIR}/usr/bin/foliate
```
Comment (sgn) [inline srcpkgs/foliate/template:17]: I prefer relative symlink for this one, looks better when inspecting filesystem from other system.

    ln -s com.github.johnfactotum.Foliate ${DESTDIR}/usr/bin/foliate

Comment (not-chicken) [reply srcpkgs/foliate/template]: I agree.

**Comment by sgn** [comment]
`./xbps-src check` failed.

**Comment by not-chicken** [comment]
> `./xbps-src check` failed.

Fails due to invalid styling in appdata file. I have contacted upstream about it and marked this PR `[WIP]` for time being.

**Comment by not-chicken** [comment]
The file has been fixed upstream. Should I create a patch? 
To be noted that this does not have any effect on the usability of the package.

**Comment by sgn** [comment]
On 2020-04-21 23:27:01-0700, Lorem <notifications@github.com> wrote:
> The file has been fixed upstream. Should I create a patch? 
> To be noted that this does not have any effect on the usability of the package.

I don't think it's matter much.
Thanks for looking into it.

One issue, though.

Please rebase against latest master.
And check which `python_version` should this package set.
I think it's safer to use python3, but I haven't tested it, yet.

-- 
Danh

**Comment by not-chicken** [comment]
> check which `python_version` should this package set.

As this [commit](https://github.com/johnfactotum/foliate/commit/ad3ec6fe243c05fd7c38e9316eb72e77400afcdf) is pushed after the release, I think it's gonna by python2 for now.

**Pull request closed without merging**